features/index/index_as_table.feature

```diff
@@ -289,3 +289,24 @@ Feature: Index as Table
       | Id | Title        | Title Length |
       | 1 | Hello World   | 11 |
       | 2 | Bye bye world | 13 |
+
+  Scenario: Show footer row
+    Given a post with the title "Hello World" exists
+    And a post with the title "Bye bye world" exists
+    And an index configuration of:
+      """
+        ActiveAdmin.register Post do
+
+          index has_footer: true do
+            column :id, footer: 'Total:'
+            column :category, footer: ->(collection) { collection.pluck(:id).sum }
```

Comment:
Hi @Fivell , @deivid-rodriguez!

I think, we should separate problem for 2 parts.

First: footer rendering. Simplest use case - it render some custom text or calculation results. This PR allow it. It can be ugly, but it works. 
I know only one other workaround: rendering entire table as ERB partial (with headers, sorting, etc).

Second: tools for calculation totals for variable situations. It may be simple or complicated. I have no tools for any cases. Developer should generate some text for footer and get valid text for rendering in his case.

- Total calculation can be simple for very small tables. In this case developer can ignore additional SQL call, if it run fast. Code is simple for writing and supporting. It may cover 40% or 80% of all situations.
- Calculation can take long time (see my example below). Developer should make it effective. 

In both cases developer should have tool to render text in footer.

My using of this PR is:

In panel DSL:

```Ruby
    panel 'Sdk stats' do
      table_for SdkStat.group_by_user do
        column(:user) do |row|
          row.user.try(:display_name)
        end
        column :total_count, footer: ->(collection) { collection.sum(&:total_count) }
        column :win_percent, footer: ->(collection) do
          MyHelper.get_percent(collection.sum(&:win_count), collection.sum(&:total_count)).round(2)
        end
      end
    end
```
It don't execute additional SQL and calculate totals in loaded collection.

In separate page (report with filters) I disable pagination:

```Ruby
  config.paginate = false

  controller do
    # returns array (engine give error without it)
    def find_collection(options = {})
      Kaminari.paginate_array(super(options), limit: 1000000, offset: 0)
    end
  end

  index pagination_total: false, has_footer: true do |users|
    column :dsp, footer: 'Итого:' do |data|
      data.user&.name
    end
    column :total_count, sortable: false, footer: ->(collection) { collection.sum(&:total_count) }
  end
```

I have no universal solution for paginated resources. I have disabled pagination in my case. But ever disabled, it try calculate totals, and I make workaround and redefine `controller#find_collection`. I have no time for fix it now, sorry.

In my old project I render 2 footer row: totals for current page and totals for all pages. But I don't use Activeadmin for this.